bus-mapping/src/evm/opcodes/ids.rs

```diff
@@ -477,6 +477,7 @@ impl FromStr for OpcodeId {
             "STATICCALL" => OpcodeId::STATICCALL,
             "SELFDESTRUCT" => OpcodeId::SELFDESTRUCT,
             "CHAINID" => OpcodeId::CHAINID,
+            "BASEFEE" => OpcodeId::BASEFEE,
```

Comment:
@han0110 correct me if I'm wrong. I remember we didn't add `BASEFEE` because we decided not to implement it too early. The more specific reason is that `BASEFEE` makes sense in the L1 validity proof use case but not the zkrollup use case.

Reply:
Yeah, you are right. And how to support EIP1559 in circuit is not decided yet. (we can implement 2 version for each use case, or single version but turn off EIP1559 in zkrollup use case)

For here I think it's missed in `FromStr`, but it's still included in previous `OpcodeId`

Reply:
Per offline discussion, adding the missing `FromStr` method can prevent Rust's panic in parsing.